### PR TITLE
Update header.html

### PR DESCRIPTION
**Pull request by ratebeerjoet**
trying regular post for Discourse header to solve search issues

### CODE REVIEW

File: common/header.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     document.getElementById('lnkLogo').setAttribute('href', `//${window.location.host.replace('community.', 'www.')}`)
   </script>
   <div data-css-w1hgs3="">
-    <form id="phoenixHeaderForm" autocomplete="off" data-css-ypxlhu="" action=".">
+    <form id="phoenixHeaderForm" autocomplete="off" action="/search">
```
Comment (shockdesign) [inline common/header.html]: @ratebeerjoet This is a relative path, it might not work when users drill down through the forums?

Comment (shockdesign) [reply common/header.html]: Should probably be something like `https://www.ratebeer.com/search` ?

Comment (ratebeerjoet) [reply common/header.html]: / means root, absolute path, whereas ../search would be relative. We could enforce the correct URL by using https://www.ratebeer.com/search if we must

Comment (ratebeerjoet) [reply common/header.html]: Something I'm uncertain of is if this page does any processing to the search query terms before it's sent to the search service. I don't think it does. In any case, if javascript doesn't like the change we can edit out of change the id=

**Pull request merged**